.travis.yml

```diff
@@ -1,6 +1,6 @@
 
 language: objective-c
-osx_image: xcode9.1
+osx_image: xcode9.3beta

```

Comment:
We can remove `beta` now.

Reply:
@rivera-ernesto 

Thanks for your comment.
Xcode 9.3 image seems to have been released immediately after I sent this pull request😭
Then I will send one more pull request later.